pkg/api/router.go

```diff
@@ -119,16 +121,32 @@ func GenerateRouter(apiConf *Config, client *pgclient.Client, elector *util.Elec
 
 func RegisterMetricsForTelemetry(t telemetry.Engine) error {
 	var err error
-	if err = t.RegisterMetric("promscale_ingested_samples_total", metrics.IngestedSamples); err != nil {
+	if err = t.RegisterMetric(
+		"promscale_ingested_samples_total",
```

Comment:
This should be removed as per design doc. Can't we move to new metrics in telemetry too, especially considering that this is only for our internal use? 

cc @cevian

Reply:
Let's do some other PR.